File: Alderto.Web/ClientApp/src/app/models/guild.ts
export interface IGuild {
  icon: string;
  id: string;
  name: string;
  owner: boolean;
  permissions: number;
}

export interface IGuildChannel {
  id: string;
  name: string;
}

export interface IGuildRole {
  id: string;
  name: string;
}

File: Alderto.Web/Models/Discord/Guild.cs
namespace Alderto.Web.Models.Discord
{
    public class Guild
    {
        public bool Owner { get; set; }
        public ulong Permissions { get; set; }
        public string Icon { get; set; }
        public ulong Id { get; set; }
        public string Name { get; set; }
    }
}